### DISCUSSION TITLE
241120_수

### DISCUSSION THREAD

**Discussion by sejongmin** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#35] [TIL]
- [x] SAM2-UNet 코드 리펙토링
- [x] wandb 추가하기

**Comment by Yoon0717** [comment]
- [x] wandb 이해 및 구축 -> 세종's wandb_logger.py 활용
- [ ] backbone model research
- [x] pixel 당 class 수 확인

**Reply by minrongtic** [reply]
하고싶은 것 -> 백본 찾기 
하기싫은 것 -> wandb

**Comment by wonjeongjeong** [comment]
- [x] 과제 2, 3
- [ ] model search 및 실험

**Comment by june21a** [comment]
- [x] segformer 성능 끌어올리기
- [x] 다른 모델 및 라이브러리 탐색
- [ ] 실험할 내용 및 모델 정리

**Comment by Soy17** [comment]
- [x] UNet++ 오류 바로 잡기
- [ ] 실험 오늘 2개 이상 돌려보기 !!
- [ ] 과제 마무리

**Comment by minrongtic** [comment]
**TO DO** 

- [ ] 23:00부터 모델 돌려보기~!
- [ ] K-fold, Stratified K-fold 공부 + 코드 짜기 -> 이슈 발행까지
- [ ] 시도해볼 백본, 스케쥴러, loss function 리스트업 하기 -> 이슈 발행/디스커션 탭/노션에 아카이빙까지 
- [ ] pre-commit 코드 포맷팅 도전~! 
- [x] 팀 노션 커버 & 아이콘 꽃단장
